src/change/controllers/changeController.ts

```diff
@@ -21,7 +22,15 @@ export interface ChangeRequestBody {
 
 @injectable()
 export class ChangeController {
-  public constructor(@inject(Services.LOGGER) private readonly logger: Logger, @inject(ChangeManager) private readonly manager: ChangeManager) {}
+  private readonly createdChangeCounter: BoundCounter;
+
+  public constructor(
+    @inject(SERVICES.LOGGER) private readonly logger: Logger,
+    @inject(ChangeManager) private readonly manager: ChangeManager,
+    @inject(SERVICES.METER) private readonly meter: Meter
+  ) {
+    this.createdChangeCounter = meter.createCounter('created_change');
```

Comment:
metrics should be calculated inside the `ChangeManager` where we handle business logic
also, `generated_changes` is clearer due to osm `create` meaning